### ISSUE TITLE
JWT Claim Check Usage

### ISSUE THREAD

**Issue by wintat** [3scale/APIcast#1201]
It is hard to follow on the JWT Claim Check Usage with limited resource

##### Version

latest

##### Steps To Reproduce

check on the configuration on https://github.com/3scale/APIcast/blob/master/gateway/src/apicast/policy/jwt_claim_check/readme.md

follow the setup with configuration 

"policy_chain":[{"name":"jwt_claim_check","version":"builtin","configuration":{"rules":[{"resource_type":"plain","operations":[{"op":"==","jwt_claim_type":"plain","jwt_claim":"admin","value_type":"plain","value":"role"}],"combine_op":"and","resource":"/resource","methods":["GET"]}]}},{"name":"apicast","version":"builtin","configuration":{}}]

putting below token in header with Authorization: Bearer 

eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZG1pbiI6InJvbGUifQ.BRdVYhLJb8fj0FIqHGoeEQzjIfNJ-DABvybbiB796Qk

##### Current Result

Response shown Request blocked due to JWT claim policy

##### Expected Result

Return the backend API response

**Comment by eloycoto** [comment]
Could you share the logs? I made a test and everything works as expected. 

Regards

**Comment by wintat** [comment]
Sorry, I am not quit sure how to get the log you need, I could just give you my configuration.
May be you could share yours configuration?
I mainly found the resources provided seems could not enable me to see the whole picture. 
Thanks. 
![01](https://user-images.githubusercontent.com/19298496/86692088-0acd0300-c03c-11ea-8cfb-79abeadb9152.png)
![02](https://user-images.githubusercontent.com/19298496/86692096-0bfe3000-c03c-11ea-82be-0f12508012f4.PNG)
![03](https://user-images.githubusercontent.com/19298496/86692097-0c96c680-c03c-11ea-8b02-24debbaa92f4.PNG)
![04](https://user-images.githubusercontent.com/19298496/86692099-0d2f5d00-c03c-11ea-84df-3ccfb9c3b59e.PNG)

**Comment by tkan145** [comment]
Closing this issue, marking it as "Stale"
